### ISSUE TITLE
When the user's focus moves to any button after the input field, the screen reader reads the entire dialog context.

### ISSUE THREAD

**Issue by smritisinha1115** [FreedomScientific/VFO-standards-support#913]
Summary:
 When the dialog opens, if a user navigates to any button after the input field using the Tab or Shift + Tab keys, the screen reader reads the entire dialog context instead of just the button label. When multiple buttons are present consecutively, the screen reader does not read the entire dialog content.

Steps to reproduce:
1.	Open the dialog box which contains input form fields on any browser.
2.	Use tab or shift + tab key to navigate the dialog.
3.	Press tabor shift+tab key until the focus on any button after the input field.
4.	Observe the screen reader announcement
5. 
Expected Behavior:
 When the dialog opens, and the user navigates to any button after the input field using the Tab or Shift + Tab keys, the screen reader should announce only the button label (and any relevant accessible name or description), not the entire dialog context.

Actual Behavior:
 When the dialog opens, if a user navigates to any button after the input field using the Tab or Shift + Tab keys, the screen reader reads the entire dialog context instead of just the button label.

Environments:
•	JAWS version:  2025.2504.89
•	Browser: Chrome, Edge, Firefox
•	OS: Windows 11
•	Assistive Technology: JAWS screen reader

Impact:
This issue can cause confusion and cognitive overload for screen reader users, as hearing the entire dialog context each time they navigate to a button makes it difficult to efficiently interact with the dialog. It may slow down navigation, increase frustration, and make it harder for users to understand which button is currently focused, potentially leading to errors or abandonment of the task.

Notes:
•	This behavior worked in previous JAWS versions.
•	Verified issue across multiple web browser. 

Attachments: 
JAWS version 2025.2504.89 behaviour:
https://github.com/user-attachments/assets/c92ef7c7-5f1a-4238-b24b-1ae09df71166
Old version of JAWS behaviour:
https://github.com/user-attachments/assets/d2232ae3-a6d0-46e9-a84a-071b8eabedab

**Comment by JeremiahRogersVispero** [comment]
I cannot replicate this behavior using [this example](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/) from the ARIA Authoring Practices Guide (APG) and JAWS 2025.2506.170. @smritisinha1115, can you provide a code sample please?

**Comment by smritisinha1115** [comment]
> I cannot replicate this behavior using [this example](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/) from the ARIA Authoring Practices Guide (APG) and JAWS 2025.2506.170. [@smritisinha1115](https://github.com/smritisinha1115), can you provide a code sample please?

Yes, I have attached the W3C Dialog example. When I press the Tab key to move focus to the "Verify Address" button, the screen reader first announces the dialog name, followed by the button name. After that, when I continue tabbing, the "Add" and "Cancel" buttons are announced as expected. However, when I use Shift+ Tab to move focus from the "Street Address" input field back to the "Cancel" button, the screen reader again announces the dialog name first, then the "Cancel" button.

Attachment of W3C Dialog example: 
https://github.com/user-attachments/assets/db754c4a-eef8-46a5-be43-b7592458db3c

**Comment by JeremiahRogersVispero** [comment]
Thanks for the quick response, @smritisinha1115. Can you upgrade to the latest public JAWS build, released in early July, available [here,](https://support.freedomscientific.com/Downloads/JAWS) and see if the behavior you are experiencing persists? If you still think the issue exists, rather than sending a video, please send code or a link to code where you can reproduce the issue. Thanks.

**Comment by JAWS-test** [comment]
I don't have this problem with the current version of JAWS.
